### PR TITLE
Support running dpdk tests on secure boot environment

### PR DESCRIPTION
**Pull request by SchSeba**
This commit continue the work done in the sriov operator vendored tests
to support secure boot enabled environments.

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: cnf-tests/testsuites/e2esuite/sctp/sctp_sriov.go [hunk 1]
```diff
@@ -39,6 +39,9 @@ var _ = Describe("[sriov] SCTP integration", func() {
 		err := namespaces.Create(TestNamespace, client.Client)
 		Expect(err).ToNot(HaveOccurred())
 
+		err = namespaces.Create(sriovnamespaces.Test, client.Client)
```
Comment (fedepaol) [inline cnf-tests/testsuites/e2esuite/sctp/sctp_sriov.go:42]: Why you need this here?

Comment (SchSeba) [reply cnf-tests/testsuites/e2esuite/sctp/sctp_sriov.go:42]: we need this namespace to exist before we run `sriovcluster.DiscoverSriov` as it start a pod on that namespace to read the secure-boot status from the node

Comment (fedepaol) [reply cnf-tests/testsuites/e2esuite/sctp/sctp_sriov.go:42]: mind adding a comment for that?

**Comment by SchSeba** [comment]
/retest

prow issue
```
ClusterOperators:
	clusteroperators are missing
error running backup collection: Get "https://api.ci-op-y6ky1ftn-26e3d.*********************************:6443/api?timeout=32s": dial tcp 34.123.186.82:6443: i/o timeoutUnable to connect to the server: dial tcp 34.123.186.82:6443: i/o timeout
{"component":"entrypoint","error":"wrapped process failed: exit status 1","file":"prow/entrypoint/run.go:80","func":"k8s.io/test-infra/prow/entrypoint.Options.Run","level":"error","msg":"Error executing test process","severity":"error","time":"2021-07-01T10:28:40Z"}
error: failed to execute wrapped command: exit status 1
```

**Comment by SchSeba** [comment]
error not related to the change

/override ci/prow/e2e-gcp

**Pull request merged**